src/osvr/PluginHost/RegistrationContext.cpp

```diff
@@ -77,12 +80,13 @@ namespace pluginhost {
                                         std::string const &name,
                                         OSVR_PluginRegContext ctx,
                                         bool shouldRethrow = false) {
-        OSVR_DEV_VERBOSE("Trying to load a plugin with the name " << name);
+        auto log = util::log::make_logger("PluginHost");
+        log->debug() << "Trying to load a plugin with the name " << name;
```

Comment:
The debug messages such as this one show up both in the cmd line output as well as log files. I think that in the log files, it is fine, however we should not be printing so many debug messages because it makes it super hard to read and might overwhelm beginners (just info and errors)
Is it possible to set a log level part of constructor that would only log debug messages to files and skip cout? or another option, if the dev wants to see those messages in cout then specify in the config file such as "log_level : debug" (not sure if this option is possible with the current implementation)